templates/microstack/docs/index.html

```diff
@@ -0,0 +1,213 @@
+---
+layout: base
+title: "Documentation"
+description: "How to install, run and configure microstack."
+---
+
+<main id="main-content">
+  <section class="p-strip--light">
+    <div class="row">
+      <div class="col-8">
+        <h1>MicroStack documentation</h1>
+        <p>OpenStack in a snap that you can run locally on a single machine!</p>
+        <p>
+          <code>microstack</code> currently provides Nova, Keystone, Glance, Horizon and Neutron OpenStack services.
+        </p>
+      </div>
+    </div>
+  </section>
+  <section class="p-strip is-bordered">
```

Comment:
Please have a look at the Rule component's documentation here: https://vanillaframework.io/docs/patterns/rule. Scroll down to Deprecated section and fix all the places in this file accordingly.

Reply:
Unnecessary redundant/old file. Removed!